recipe/meta.yaml

```diff
@@ -10,20 +10,30 @@ source:
   sha256: b7287480a1f6ae4b6fc80d775446d8af00e051ca1646b6cc3d32c5d3a461ede3
 
 build:
-  number: 1
-  skip: true  # [not win]
+  number: 2
 
 requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('cxx') }}  # [not win]
+    - clangdev  # [linux]
+    - pkg-config  # [not win]
   host:
+    - mono  # [not win]
     - pip
     - pycparser
     - python
+    - wheel
   run:
+    - mono  # [not win]
     - python
 
 test:
   imports:
-    - clr
+    - clr  # [not linux]
+  commands:
+    - export LD_LIBRARY_PATH=${BUILD_PREFIX}/x86_64-conda-linux-gnu/sysroot/usr/lib/  # [linux]
```

Comment:
Otherwise the import tests fail, because `libdl.so` cannot be found. Pythonnet does it in [a similar way](https://github.com/pythonnet/pythonnet/blob/2ad4e7006a3646be8457f74940541812e79a926e/.travis.yml#L40) and [we did in the past](https://github.com/conda-forge/pythonnet-feedstock/blob/25706609f26ed1f03cd68abf0478297d0c605198/recipe/meta.yaml#L36), too